modules/oe_content_call_proposals/src/CallForProposalsNodeWrapper.php

```diff
@@ -0,0 +1,204 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_content_call_proposals;
+
+use Drupal\Component\Render\MarkupInterface;
+use Drupal\Core\Datetime\DrupalDateTime;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+use Drupal\oe_content\EntityWrapperBase;
+
+/**
+ * Wrap the "Call for proposals" content type by adding business methods.
+ *
+ * @method static CallForProposalsNodeWrapperInterface getInstance(\Drupal\Core\Entity\ContentEntityInterface $entity)
+ */
+class CallForProposalsNodeWrapper extends EntityWrapperBase implements CallForProposalsNodeWrapperInterface {
+
+  use StringTranslationTrait;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getEntityId(): string {
+    return 'node';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getEntityBundle(): string {
+    return 'oe_call_proposals';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function hasOpeningDate(): bool {
+    return !$this->entity->get('oe_call_proposals_opening_date')->isEmpty();
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function hasDeadlineDate(): bool {
+    return !$this->entity->get('oe_call_proposals_deadline')->isEmpty();
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getOpeningDate(): ?DrupalDateTime {
+    if (!$this->hasOpeningDate()) {
+      return NULL;
+    }
+    $opening_date = $this->entity->get('oe_call_proposals_opening_date')->date;
+    // Prevent upcoming status when now & opening dates are the same.
+    $opening_date->setTime(0, 0, 0);
+    return $opening_date;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getDeadlineDate(): ?DrupalDateTime {
+    if (!$this->hasDeadlineDate()) {
+      return NULL;
+    }
+
+    $dates = $this->entity->get('oe_call_proposals_deadline')->getValue();
+
+    if (!is_array($dates)) {
+      return NULL;
+    }
+
+    // Get the latest date in relation to the "closed" proposal's status.
+    array_multisort(array_column($dates, 'value'), SORT_DESC, $dates);
```

Comment:
Only variables can be passed by reference to array_multisort()

Reply:
This is fixed.